tests/integration/test_annotations.py

```diff
@@ -0,0 +1,37 @@
+import os
+
+from click.testing import CliRunner
+from pytest import fail
+
+from tests import fixtures_dir
+from tests import root
+from xsdata.cli import cli
+from xsdata.formats.dataclass.context import XmlContext
+from xsdata.formats.dataclass.parsers.xml import XmlParser
+from xsdata.utils.testing import load_class
+
+os.chdir(root)
+
+
+def test_annotations():
+    filepath = fixtures_dir.joinpath("annotations")
+    schema = filepath.joinpath("model.xsd")
+    runner = CliRunner()
+    result = runner.invoke(
+        cli, [str(schema), f"--config={str(filepath.joinpath('xsdata.xml'))}"]
```

Comment:
All GeneratorOutput options appear in the cli automatically, no need to add a new config file.

```
❯ xsdata generate --help
...
...
...

Options:
  -r, --recursive                 Search files recursively in the source
                                  directory
  -c, --config TEXT               Project configuration
  -pp, --print                    Print output
  -p, --package TEXT              Target package, default: generated
  -o, --output [dataclasses]      Output format name, default: dataclasses
  --repr / --no-repr              Generate __repr__ method, default: true
  --eq / --no-eq                  Generate __eq__ method, default: true
...
...
...
...
  --import-annotations / --no-import-annotations
                                  Import annotations from :obj:`__future__` in
                                  generated modules, default: false
  --help                          Show this message and exit.

```

Reply:
And fix the help text as well, it comes fromt the docstring

Reply:
Is it possible to document it in such a way that the HTML docs have the link for the `__future__` module, but the CLI help is only text ?

Reply:
Maybe there is no need to have implementations details in the docstring. I think the following should be sufficient.

```python
:param postponed_annotations: Enable Postponed Evaluation of Annotations, default: false
```